index.html

```diff
@@ -59,6 +59,170 @@ <h2 class="primary-header green">
+      <h1 class="primary-header">
+        My Recent Works
+        <p id="indicator"></p>
+      </h1>
+      <div class="works-cards">
+        <article class="card">
+          <header class="card-header"></header>
+          <div class="card-content">
+            <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+            <p class="project-tags">
+  
+            <ul class="project-tags-list">
+              <li><button class="tag-button">Ruby on rails</button></li>
+              <li><button class="tag-button">css</button></li>
+              <li><button class="tag-button">Javascript</button></li>
+              <li><button class="tag-button">html</button></li>
+            </ul>
+            </p>
+  
+            <button class="green-button" id="detail-button">See Project</button>
+          </div>         
+        </article>
+  
+        <article class="card">
+          <header class="card-header"></header>
+          <div class="card-content">
+            <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+            <p class="project-tags">
+  
+            <ul class="project-tags-list">
+              <li><button class="tag-button">Ruby on rails</button></li>
+              <li><button class="tag-button">css</button></li>
+              <li><button class="tag-button">Javascript</button></li>
+              <li><button class="tag-button">html</button></li>
+            </ul>
+            </p>
+  
+            <button class="green-button" id="detail-button">See Project</button>
+          </div>          
+        </article>
+  
+        <article class="card">
+          <header class="card-header"></header>
+          <div class="card-content">
+            <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+            <p class="project-tags">
+  
+            <ul class="project-tags-list">
+              <li><button class="tag-button">Ruby on rails</button></li>
+              <li><button class="tag-button">css</button></li>
+              <li><button class="tag-button">Javascript</button></li>
+              <li><button class="tag-button">html</button></li>
+            </ul>
+            </p>
+  
+            <button class="green-button" id="detail-button">See Project</button>
+          </div>          
+        </article>
+  
+        <article class="card">
+          <header class="card-header"></header>
+          <div class="card-content">
+            <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+            <p class="project-tags">
+  
+            <ul class="project-tags-list">
+              <li><button class="tag-button">Ruby on rails</button></li>
+              <li><button class="tag-button">css</button></li>
+              <li><button class="tag-button">Javascript</button></li>
+              <li><button class="tag-button">html</button></li>
+            </ul>
+            </p>
+  
+            <button class="green-button" id="detail-button">See Project</button>
```

Comment:
- [ ] kindly add the button interactions styling: Active, Hover, and Pressed like mentioned in Figma design , please add it to all of your buttons

![image](https://user-images.githubusercontent.com/89282221/193311139-1bf5d651-bc2e-4949-a1e8-85f484d6e701.png)

Reply:
Done